backend/src/graphql.ts

```diff
@@ -150,12 +150,16 @@ schema.queryType({
     t.list.field('posts', {
       type: 'Post',
       args: {
+        status: arg({ type: 'PostStatus', required: false }),
```

Comment:
`PostStatus` should be `required: true` 👍🏼 (every post should have a status and we should always pass the one we want here)